### ISSUE TITLE
Use a public CDN endpoint to get MaxMind GeoLite2 Data by default

### ISSUE THREAD

**Issue by asifsmohammed** [opensearch-project/data-prepper#3942]
Use a public CDN endpoint to improve the user experience and provide MaxMind GeoLite2 Database by default. 

Configuration for using geoip can be as simple as this,
```
processor:
  - geoip:
      keys:
          - key:
              source: "/peer/ip"
              target: "target1"
```

**Comment by dlvenable** [comment]
@asifsmohammed , Do you have a configuration proposal for the `data-prepper-config.yaml` file?

Also, a CDN is a server-side technology to push content close to users. Clients experience this as making a request to a URL. So I'd rephrase this proposal to "Use a configurable HTTP endpoint to get..."
